Dockerfile_publish

```diff
@@ -42,8 +46,4 @@ RUN pip3 install --break-system-packages --upgrade "/code[gcp]" \
 
 EXPOSE 9912
 
-RUN apt-get satisfy --yes --no-install-recommends \
```

Comment:
pretty sure these are actually required so that the publisher can call out to gpg (for gpgme)